### PR TITLE
[FIX] Broken settings check for updates

### PR DESCRIPTION
**Pull request by geekgonecrazy**
## Proposed changes (including videos or screenshots)

If someone choses not to register but still want to find out about new versions, or critical vulnerabilities this code would never fire.

### CODE REVIEW

File: apps/meteor/app/version-check/server/index.ts [hunk 1]
```diff
@@ -24,14 +24,14 @@ const addVersionCheckJob = Meteor.bindEnvironment(() => {
...
-		if (settings.get('Register_Server') && settings.get('Update_EnableChecker')) {
+		if (settings.get('Update_EnableChecker')) {
 			Promise.await(checkVersionUpdate());
 		}
 	});
 });
 
-settings.watchMultiple(['Register_Server', 'Update_EnableChecker'], () => {
-	const checkForUpdates = settings.get('Register_Server') && settings.get('Update_EnableChecker');
+settings.watchMultiple(['Update_EnableChecker'], () => {
```
Comment (debdutdeb) [inline apps/meteor/app/version-check/server/index.ts:33]:
```suggestion
settings.watch('Update_EnableChecker', () => {
```

**Review by debdutdeb — CHANGES_REQUESTED** [review]

**Review by debdutdeb — APPROVED** [review]
makes sense to me :)

**Review by KevLehman — APPROVED** [review]
Should this have any form of QA?

**Comment by debdutdeb** [comment]
I honestly don't see the need for this patch 😬 idk

**Pull request merged**